lib/taglessFinal/dune

```diff
@@ -20,18 +20,17 @@
       11.0
       (progn
        (run ./configure)
-       (run make libpython3.11.a))))
-    (copy vendor/libpython3.11.a libpython.a)
```

Comment:
I think that it's likely easier to just create a static library here and statically link it to both the static and the dynamic binding of taglessFinal (ie the artefact of binding.c)